components/higher-order/with-api-data/request.js

```diff
@@ -80,12 +60,12 @@ export function getCachedResponse( request ) {
 }
 
 export function getResponseFromNetwork( request ) {
-	const promise = apiRequest( request )
-		.then( ( body, status, xhr ) => {
-			return {
-				body,
-				headers: getResponseHeaders( xhr ),
-			};
+	const promise = apiFetch( { ...request, parse: false } )
+		.then( ( response ) => {
+			return response.json().then( ( body ) => ( {
+				body: body,
+				headers: response.headers,
```

Comment:
What is the shape of `response.headers` here ? Is it an object implementing the [`Headers` interface](https://developer.mozilla.org/en-US/docs/Web/API/Headers) ? That doesn't seem compatible with how this was previously shaped as a set of tuples.

Reply:
unless I'm missing something, I don't see it being used or exposed anywhere. I think it should be fine to remove entirely.